lib/rbac/filterer.rb

```diff
@@ -797,7 +797,9 @@ def get_belongsto_matches(blist, klass)
     def belongsto_association_filtered?(vcmeta, klass)
       if [ExtManagementSystem, Host].any? { |x| vcmeta.kind_of?(x) }
         # Eject early if true
```

Comment:
while your at it...

Could you change this to "eject early if the models do not ..."

eject early if true is useless. and this code is a little confusing so it would be nice to know what it does in english

Reply:
sure, can you check whether comment explains it more now ? 

thanks!

Reply:
Who was the a--hat who wrote that comment!!1!

Oh it was me... 😞